atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/ConnectionConfig.java

```diff
@@ -114,10 +114,9 @@ public String getConnectionPoolIdentifier() {
     @Value.Derived
     public String getConnectionPoolName() {
         return String.format(
-                "%s-%s-%s",
+                "%s-%s",
                 getConnectionPoolIdentifier(),
-                getConnId(),
-                getDbLogin());
+                getConnId());
```

Comment:
Could we use a tritium tagged metric registry + tagged MetricName for hikari pools? the connId and pool identifier would be great tag values on a single `database.pool` metric name (example name was the first thing I though of, I'm sure there's a better name for it)

Reply:
I'm not sure if this is possible, because I think the metrics are just coming from an MBean from Hikari (the name we set here I think becomes the name of the MBean)

Reply:
Actually, it looks like we can make an improvement here, Hikari has a PrometheusMetricsTracker that should be able to register prometheus metrics directly (the pool name becomes a label on the metrics instead of being part of the metric name). If we want some custom tagged metrics registry we could probably implement our own MetricsTrackerFactory

Reply:
I'd prefer to wire it directly to the tritium registry, which can be connected to prometheus as well as sls-logging metrics pipelines.

Reply:
To do that we'd need to make a custom MetricsTrackerFactory. I think it would probably be better to do that in a separate change.